Code/Engine/Foundation/Math/Color.h

```diff
@@ -246,6 +246,7 @@ class EZ_FOUNDATION_DLL ezColor
   /// \brief Sets all four RGBA components.
   void SetRGBA(float fLinearRed, float fLinearGreen, float fLinearBlue, float fLinearAlpha = 1.0f); // [tested]
 
+  static ezColor MakeFromKelvin(ezUInt32 uiKelvin);
```

Comment:
Can you also add a comment to this function?

Reply:
Added